packages/core/contracts/common/implementation/AncillaryData.sol

```diff
@@ -92,15 +92,16 @@ library AncillaryData {
         bytes memory key,
         uint256 value
     ) internal pure returns (bytes memory) {
-        bytes memory prefix = _appendKey(currentAncillaryData, key);
+        bytes memory prefix = constructPrefix(currentAncillaryData, key);
         return abi.encodePacked(currentAncillaryData, prefix, toUtf8BytesUint(value));
     }
 
     /**
-     * @notice Helper method that returns the LHS of a "key:value" pair plus the colon ":" and a leading comma "," if
-     * the ancillary data to append to is not empty.
+     * @notice Helper method that returns the left hand side of a "key:value" pair plus the colon ":" and a leading
+     * comma "," if the `currentAncillaryData` is not empty. The return value is intended to be appended as a prefix to
+     * some utf8 value that is ultimately added to a comma-delimited, key-value dictionary.
      */
-    function _appendKey(bytes memory currentAncillaryData, bytes memory key) internal pure returns (bytes memory) {
```

Comment:
- `AncillaryData` employs an inconsistent underscore prefix for internal functions. Example: [`_appendKey`](https://github.com/UMAprotocol/protocol/blob/a3bf46270787cbaae4ed2218f064b1217c153a50/packages/core/contracts/common/implementation/AncillaryData.sol#L103) and [`appendKeyValueUint`]